build.proj

```diff
@@ -258,6 +258,8 @@
         ContinueOnError="false" 
         Condition="!$(DelaySign) and '@(DelaySignedAssembliesToSign)' != ''"/>
         
+    <Exec Command="$(MSBuildProjectDirectory)\tools\CheckStrongNameSignature.ps1"/>
```

Comment:
See: https://github.com/cormacpayne/azure-powershell/blob/5043c685511c5fc8d678a4e245c9bdfa41ad83ff/build.proj#L308  for how to run powershell and refer to the tools folder in a task